Resources/Textures/_Nuclear14/Clothing/Mask/Gasmask/zealotgasmask.rsi/meta.json

```diff
@@ -0,0 +1,26 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "CT-56324864",
```

Comment:
Similarly, you may want to add "Created by X on Discord" in addition to any other info they have like a github username if they have it.